charts/sysdig/templates/configmap-image-analyzer.yaml

```diff
@@ -46,17 +44,15 @@ data:
   no_proxy: {{ .Values.nodeImageAnalyzer.settings.noProxy }}
   {{- end -}}
 {{- end }}
-{{- if and (not .Values.nodeImageAnalyzer.deploy) .Values.nodeAnalyzer.deploy}}
+{{- if and (not (include "deploy-nia" .)) .Values.nodeAnalyzer.deploy}}
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ template "sysdig.fullname" . }}-image-analyzer
   labels:
 {{ include "sysdig.labels" . | indent 4 }}
 data:
-  {{- if .Values.nodeAnalyzer.collectorEndpoint }}
-  collector_endpoint: {{ .Values.nodeAnalyzer.collectorEndpoint }}
```

Comment:
This was wrong, as https:// and the suffix is added in the other configmaps, but it was missing in here.